conda-recipe/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: covid-if-annotations
+  version: 0.0.3dev1
+
+source:
+  path: ..
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  entry_points:
+    {% for ep in data['entry_points']['console_scripts'] %}
+    - {{ ep }}
+    {% endfor %}
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    # can be removed if bumpversion PR is merged
+    - setuptools_scm
```

Comment:
I just merged the bumpversion PR, so can be removed :)

Reply:
done by rebasing.